functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/Project.java

```diff
@@ -36,6 +42,32 @@ public ProjectConfigurator configureSmartTesting() {
         return new ChangeApplier(repository).applyAsCommits(changeDescriptions);
     }
 
+    Project checkoutBranch(String branchName) {
+        try {
+            git.checkout().setName(branchName).call();
```

Comment:
well at that time we had two options:

* manipulate POM 
*  do this change in the repo directly, with the new branch

we agreed to do the second one, for this reason this is done in that way, saying that I am more than open to change it because I suspect there are some problems with this approach.